src/Compilers/CSharp/Test/Emit3/Semantics/CollectionExpressionTests_WithElement_Constructor.cs

```diff
@@ -1299,11 +1298,10 @@ void M()
             }
             """;
 
-        // PROTOTYPE: This error message isn't good.  We should say that the constructor is inaccessible.
```

Comment:
The change to return an inaccessible constructor if we find no accessible constructor in the 'with' case is what resolved this.